app/views/articles/index.html.erb

```diff
@@ -11,3 +11,5 @@
     </li>
   <% end %>
 </ul>
+
+<%= paginate @articles %>
```

Comment:
You still need to set up your editor to automatically add newlines to the end of files.